PERSONAL_DATA_DISCLOSURE.md

```diff
@@ -1,58 +1,74 @@
-
-
 ## Overview
-The purpose of this form is to disclose the types of personal data stored by each module.  This information enables those hosting FOLIO to better manage and comply with various privacy laws and restrictions, e.g. GDPR.
 
-It's important to note that personal data is not limited to that which can be used to identify a person on it's own (e.g. Social security number), but also data used in conjunction with other data to identify a person (e.g. date of birth + city + gender).
+The purpose of this form is to disclose the types of personally identifiable information (PII) stored by each module.  This information enables those hosting FOLIO to better manage and comply with various privacy laws and restrictions, e.g. GDPR.
```

Comment:
Suggest ..._stored and processed_ by each module.

A module may receive PII from another module for processing and not end up storing it.  The processing aspects should also be noted in this document as well.  There is probably some GDPR terminology that could be used here.